.github/workflows/publish_v3.yml

```
@@ -32,6 +32,16 @@ jobs:
         run: pip install -r spec-parser/requirements.txt
       - name: Build model
         run: python3 spec-parser/main.py spdx-3-model/model spdx-spec/docs/model
+      - name: Copy JSON annotations
+        run: cp spdx-spec/serialization/jsonld/annotations.ttl spdx-spec/docs/model/jsonld-annotations.ttl
+      - name: Generate JSON schema
+        run: |
+          shacl2code generate \
+            --input spdx-spec/docs/model/ontology.rdf.ttl \
+            --input spdx-spec/docs/model/jsonld-annotations.ttl \
+            --context-url spdx-spec/docs/model/context.jsonld https://spdx.org/rdf/3.0.0/spdx-context.jsonld \
```

I am confused? why is there an argument before the url?
```suggestion
            --context-url https://spdx.org/rdf/3.0.0/spdx-context.jsonld \
```
Maybe I am just not aware of the API of your tool

---

The tool lets you either specify a URL (with --context) or a local file and URL (with --context-url) in which case the data is read from the local file, but the URL is used in the bindings. The later is useful to generate bindings before the URL is published